matcher/service/openreview_interface.py

```diff
@@ -90,15 +90,17 @@ def get_all_edges(client, edge_invitation_id, logger=None):
     logger.debug('GET grouped edges invitation id={}'.format(edge_invitation_id))
     for group in edges_grouped_by_paper:
         forum_id = group['id']['head']
-        for group_value in group['values']:
-            all_edges.append(build_edge(
-                edge_invitation,
-                forum_id,
-                group_value['tail'],
-                group_value.get('weight'),
-                group_value.get('label'),
-                None
-            ))
+        if forum_id in all_papers:
+            for group_value in group['values']:
+                if group_value['tail'] in all_reviewers:
+                    all_edges.append(build_edge(
```

Comment:
no need to call `build_edge` here because the interface is returning a list of tuples to the encoder.

Reply:
Fixed.